packages/gatsby-plugin-screenshot/README.md

```diff
@@ -0,0 +1,59 @@
+# gatsby-plugin-screenshot
+
+Plugin for creating screenshots of website URLs using an AWS Lambda
+Function. This plugin looks for `SitesYaml` nodes with a `url`
+property, and creates `Screenshot` nodes with an `imageFile` field. 
+
+[Live demo](https://thatotherperson.github.io/gatsby-screenshot-demo/)
+([source](https://github.com/ThatOtherPerson/gatsby-screenshot-demo))
+
+## Install
+
+`npm install gatsby-plugin-screenshot`
+
+## How to use
+
+```javascript
+// in your gatsby-config.js
+plugins: [
+  {
+    resolve: `gatsby-plugin-screenshot`,
+    options: {
+      lambdaName: `gatsby-screenshot-lambda`,
+      region: 'us-west-2',
+      credentials: {        // optional
+        accessKeyId: 'xxxx',
```

Comment:
Look at https://github.com/gatsbyjs/gatsby/pull/3526/files/b3797101f312b54492b021d874518542afb5c28c#diff-9d836096b5349fa77536e46b0dfb13f4R34 - providing credentials here is optional. You can provide credentials through the other methods AWS provides - for example, on my computer I have them in `~/.aws/credentials`.

Reply:
Sure but I mean anyone trying to run gatsbyjs.org locally now won't be able to see screenshots unless we distribute the access key, etc. It'd be better if the lambda had an associated API gateway instead of invoking the lambda directly.